bin/install-wordpress.sh

```diff
@@ -74,12 +74,6 @@ docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm $CONTAINER chmod 767 /var/w
 CURRENT_WP_VERSION=$(docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run -T --rm $CLI core version)
 echo -e $(status_message "Current WordPress version: $CURRENT_WP_VERSION...")
 
-if [ "$WP_VERSION" == "latest" ]; then
```

Comment:
If we decide to do it, we should also remove all usage, e.g.:
https://github.com/WordPress/gutenberg/blob/master/.travis.yml#L61